extensions/amp-animation/0.1/css-expr-ast.js

```diff
@@ -26,10 +27,11 @@ const INFINITY_RE = /^(infinity|infinite)$/i;
  * parsing and evaluation is heavy, but used relatively rarely. This method
  * can be used to avoid heavy parse/evaluate tasks.
  * @param {string} css
+ * @param {boolean} normalize
  * @return {boolean}
  */
-export function isVarCss(css) {
-  return VAR_CSS_RE.test(css);
+export function isVarCss(css, normalize) {
+  return VAR_CSS_RE.test(css) || normalize && NORM_CSS_RE.test(css) || false;
```

Comment:
Not needed anymore. I started with `normalize` being an optional attribute. Removed.